pkgs/development/misc/umr/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchgit, bash-completion, cmake, pkg-config
+, libdrm, libpciaccess, llvmPackages, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "umr";
+  version = "unstable-2021-02-18";
+
+  src = fetchgit {
+    url = "https://gitlab.freedesktop.org/tomstdenis/umr";
+    rev = "79e17f8f2807ed707fc1be369d0aad536f6dbc97";
+    sha256 = "IwTkHEuJ82hngPjFVIihU2rSolLBqHxQTNsP8puYPaY=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [
+    bash-completion
+    libdrm
+    libpciaccess
+    llvmPackages.llvm
+    ncurses
+  ];
+
+  # Remove static libraries (there are no dynamic libraries in there)
+  postInstall = ''
+    rm -r $out/lib
```

Comment:
Is there a build option to not compile them at all?

Reply:
No, seems like there is no such option. `umrcore` needs to be built as a dependency of `umr` and the `CMakeLists.txt` unconditionally install it.
I don’t know if there is a way to tell CMake to skip installing some targets.